UnitTests/UnitTests.UWP/Assets/Samples/InfiniteCanvasExport.json

```diff
@@ -1,7 +1,7 @@
 {
   "DrawableList": [
     {
-      "$type": "Microsoft.Toolkit.Uwp.UI.Controls.InkDrawable, Microsoft.Toolkit.Uwp.UI.Controls",
+      "$type": "Microsoft.Toolkit.Uwp.UI.Controls.InkDrawable, Microsoft.Toolkit.Uwp.UI.Controls.Media",
```

Comment:
@azchohfi talking to Rosario, this appears to be the assembly name. So this is going to break compatibility with previously exported InfiniteCanvas files, eh? This is going to be an issue for apps using this control, right?

Any thoughts on how we may be able to mitigate this? This is used directly by `System.Text.Json` right? Is there some custom Type resolver we could provide to help it do the migration?

Reply:
We already have a custom type resolver:
https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/0d43da7ae459d000ee29dcb108cc08a56bf5073f/Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/JsonConverters/IDrawableConverter.cs#L43-L50
We could either introduce a new version or just try to handle this here. I think it makes more sense to introduce a new version.

Reply:
e4db504 Added a check for the old Discriminator.